packages/server/src/integrations/postgres.ts

```diff
@@ -41,7 +41,7 @@ if (types) {
   types.setTypeParser(1184, (val: any) => val) // timestampz
 }
 
-const JSON_REGEX = /'{.*}'::json/s
+const JSON_REGEX = /'{\s*.*?\s*}'::json/gs
```

Comment:
Is this just to exclude empty `{}`? And if the `g` flag is needed, can we add a test that processes multiple matches, to ensure this is not deleted in the future?

Reply:
Its to make sure multiple matches don't get combined into one - the test case I've added with this is already checking for that (does the JSON statement twice).